testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/sse/resource/SseReconnectResource.java

```diff
@@ -18,15 +17,14 @@
 import javax.ws.rs.sse.SseEventSink;
 import java.util.concurrent.TimeUnit;
 
-@Singleton
 @Path("/reconnect")
 public class SseReconnectResource {
 
-   private volatile boolean isServiceAvailable = false;
+   private static volatile boolean isServiceAvailable = false;
 
-   private volatile long startTime = 0L;
-   private volatile long endTime = 0L;
-   private volatile long lastEventDeliveryTime;
+   private static volatile long startTime = 0L;
+   private static volatile long endTime = 0L;
+   private static volatile long lastEventDeliveryTime;
```

Comment:
These could be correct or not (I haven't checked), but are they really related to this PR / RESTEASY-2686?

Reply:
See my previous comment https://github.com/resteasy/Resteasy/pull/2505#discussion_r484677003

Reply:
OK, thanks for the explanation, makes sense (here and in the other discussion)